src/libraries/System.Net.Http/src/System/Net/Http/MultipartContent.cs

```diff
@@ -240,7 +240,7 @@ private async Task<Stream> CreateContentReadStreamAsyncCore(CancellationToken ca
                         // as we don't necessarily own them yet.
 
                         // Do not pass a cancellationToken to base as it would trigger an infinite loop => StackOverflow
-                        return await base.CreateContentReadStreamAsync().ConfigureAwait(false);
+                        return await base.CreateContentReadStreamAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
This needs to be reverted.

Reply:
Yep, the analyzer was merged as info (sorry for the late reply).

Reply:
Set to default.